chain/x/oracle/client/common/proof/multi_store.go

```diff
@@ -20,6 +20,23 @@ import (
 	"github.com/bandprotocol/bandchain/chain/x/oracle/types"
 )
 
+// MultiStoreProof stores a compacted version of other store Merkle hash to compute
```

Comment:
a compact of other Cosmos-SDK modules' storage hash in multistore to compute (in combination with oracle store hash) Tendermint's application state hash at a given block.